lib/cext/patches/byebug_patches.rb

```diff
@@ -6,6 +6,8 @@
 # GNU General Public License version 2
 # GNU Lesser General Public License version 2.1
 
+require_relative 'common_patches'
```

Comment:
Looks fine, but in what case the load order from preprocess.rb would not be enough?

Reply:
I wanted to work with a set of patches in an IRB session and not think about where I needed to load them from.